config/build/webpack.config.common.js

```diff
@@ -34,7 +35,7 @@ export const createResolveConfigLoader = () => ({
 
 export function config(options) {
     return {
-        devtool: options.dev && 'source-map',
+        devtool: options.dev && 'eval-cheap-module-source-map',
```

Comment:
Well... I guess I'm going to revert it then. The good news is that it's only a small part of the performance improvement.